src/Features/CSharp/Portable/EditAndContinue/SyntaxUtilities.cs

```diff
@@ -187,7 +187,8 @@ public static SyntaxNode TryGetEffectiveGetterBody(ArrowExpressionClauseSyntax p
                 return propertyBody.Expression;
             }
 
-            return accessorList?.Accessors.Where(a => a.IsKind(SyntaxKind.GetAccessorDeclaration)).FirstOrDefault()?.Body;
+            var firstAccessor = accessorList?.Accessors.Where(a => a.IsKind(SyntaxKind.GetAccessorDeclaration)).FirstOrDefault();
+            return (SyntaxNode)firstAccessor?.Body ?? firstAccessor?.ExpressionBody?.Expression;
```

Comment:
>?.Body ?? firstAccessor?.E [](start = 44, length = 26)

A lot of ? here. Perhaps change to 
if (firstGetter == null)
{
  return null;
}

to make it clearer